circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
+    - pyenv local 3.4.3:
+        parallel: true
+    - pip install -r docs-requirements.txt:
+        parallel: true
+    - rm -rf docs/API && make -C docs clean:
+        parallel: true
     - python setup.py docs:
         parallel: true
```

Comment:
I dont understand why you prefer to install the same python version on both nodes (it is already installed on one of them) and run Sphinx on the same Python version on two nodes.  That gives us nothing.  Just dont run it on one node.